src/js/common/schemaform/definitions/address.js

```diff
@@ -225,3 +225,289 @@ export function uiSchema(label = 'Address', useStreet3 = false, isRequired = nul
     }
   };
 }
+
+
+// PCIU Validations
+
+function validateSpecialCharacter(errors, string) {
+  if (string && !isValidSpecialCharacter(string)) {
+    errors.addError(
+      "Please only use letters, numbers, and the special characters #%&'()+,./:@"
+    );
+  }
+}
+
+function validateZIP(errors, zip) {
+  if (zip && !isValidUSZipCode(zip)) {
+    errors.addError('Please enter a valid 5 or 9 digit ZIP (dashes allowed)');
```

Comment:
It looks like there are instances of ZIP that aren't capitalized in letters, should those be?